Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1349,3 +1354,63 @@ private class BrowserScreenshotHelper: ScreenshotHelper {
         return nil
     }
 }
+
+private class BrowserPrintPageRenderer: UIPrintPageRenderer {
+    private weak var browser: Browser?
+    let textAttributes = [NSFontAttributeName: UIFont(name: "Helvetica", size: 8)!]
```

Comment:
We've got some font stuff in AppConstants that I'd prefer to use here, but its all UIFonts. You might need to split it up a bit. i.e. AppConstants.SmallFontSize and/or AppConstants.DefaultFontName and/or AppConstants.DefaultSmallFontAttrbitues